### PR TITLE
Reorganize HTML structure of site template

### LINKED ISSUE #3624: FluidFramework.com skip to main content function isn't working
1. Load fluidframework.com
2. Tab
Select skip to main content
3. Observe that focus stays on the skip to main content element.

### PR DESCRIPTION
**Pull request by tylerbutler**
- Remove unused analytics partial.
- Remove duplicate cookie script.
- Move nav bar and Skip to Content button outside the #main-content div (fixes #3624).

### CODE REVIEW

File: docs/themes/thxvscode/layouts/partials/header.html [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<!-- EU Cookie Compliance JS -->
-<script src="//uhf.microsoft.com/mscc/statics/mscc-0.3.6.min.js"></script>
\ No newline at end of file
+<a role="button" id="skip-to-content" class="link-button" href="#main-content">Skip to content<i>&nbsp;</i><span
```
Comment (micahgodbolt) [inline docs/themes/thxvscode/layouts/partials/header.html:1]: i know this wasn't added by you, but I don't understand the need for `role="button"`. This is an anchor tag that navigates you to a different part of the page. [Role button](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) is often used when a div/span needs to be clickable.

```suggestion
<a id="skip-to-content" class="link-button" href="#main-content">Skip to content<i>&nbsp;</i><span
```

Comment (micahgodbolt) [reply docs/themes/thxvscode/layouts/partials/header.html:1]: ah, we're treating it as a button by using JS to turn a click into a focus on first focusable element 

image

still feels redundant, but at least I see the logic of why someone did that

**Comment by micahgodbolt** [comment]
unless I missed something, the playground.html file needs analtyics.html removed as well (got build error first time around)

image

**Review by micahgodbolt — CHANGES_REQUESTED** [review]
ensure that playground.html isn't importing missing partial, and consider changes for API/Docs pages

File: docs/themes/thxvscode/layouts/partials/header.html (hunk 1 above)
Comment (tylerbutler) [reply docs/themes/thxvscode/layouts/partials/header.html:1]: This was probably inherited from the template we were gifted. :) Worth filing a follow up issue?

Comment (micahgodbolt) [reply docs/themes/thxvscode/layouts/partials/header.html:1]: not really a big deal. not worth the cycles to track, just a nit since it's part of this PR

**Comment by tylerbutler** [comment]
> unless I missed something, the playground.html file needs analtyics.html removed as well (got build error first time around)

Wow, not sure how I missed this!

I made more changes to the structure to accommodate customization of which element is #main-content.

**Comment by micahgodbolt** [comment]
the placement of the 'main content' div seems a little erratic. 

1. Homepage doesn't have a 'main content'
2. API home looks good
image

3. But internal API pages still have main-content wrapped around entire page

image

4. Community page is good

**Comment by micahgodbolt** [comment]
Docs looks good, though I can't say I love the "put focus on first focusable element" when that element might be the last link on the entire page. Checking it out with a screen reader

**Comment by micahgodbolt** [comment]
yeah, narrator jumps to the first link, reads it, and then does nothing. If you're using narrator to read the page, you have to travel back up the page (who knows how far) to get to the top.

**Comment by tylerbutler** [comment]
I noticed the jumping to link behavior too but I'm not sure what's causing it. Another thing we inherited. I saw the same behavior on the vscode site FWIW.

**Comment by micahgodbolt** [comment]
It's coming from this little bit of js

https://github.com/microsoft/FluidFramework/blob/9f048cda77cd7263a0068ff45fc4dd00943890b6/docs/themes/thxvscode/assets/js/jquery/accessibility.js#L1-L12

From a focus management perspective, it's ok......but it sucks for screen readers

**Comment by tylerbutler** [comment]
> the placement of the 'main content' div seems a little erratic.
> 
> 1. Homepage doesn't have a 'main content'
> 2. API home looks good
> 3. But internal API pages still have main-content wrapped around entire page
> 4. Community page is good

Homepage now has main-content:

image

And API pages now wrap main-content around the in-page nav but not the cross-page nav.

**Comment by micahgodbolt** [comment]
looks good. Still would love to reconsider that accessibility.js sometime, but for now this is much better than what we had before.

**Review by micahgodbolt — APPROVED** [review]

**Comment by curtisman** [comment]
Close and reopening to trigger the pipeline

**Pull request merged**